plugins/accuracy.cc

```diff
@@ -600,7 +600,34 @@ run(const ProcedureRunConfig & run,
     // 1.  Get the input dataset
     SqlExpressionMldbScope context(server);
 
-    auto dataset = runAccuracyConf.testingData.stm->from->bind(context).dataset;
+    auto boundDataset = runAccuracyConf.testingData.stm->from->bind(context);
```

Comment:
On veut pas plus le faire comme c'est fait ici? https://github.com/mldbai/mldb/blob/657d43763af39c8f3f0fb138111fce98fe869290/plugins/accuracy.cc#L509

Genre chacune des 3 méthodes d'évaluation doit déjà faire le select, le where, etc, et itère et a le compte...